Makefile

```
@@ -24,6 +39,13 @@ $(SAMPLEDATA):
 
 sampledata: $(SAMPLEDATA)
 
+bin/$(ZEEKPATH):
+	@mkdir -p bin
+	@curl -L -o bin/$(ZEEKPATH).zip \
+		https://github.com/brimsec/zeek/releases/download/$(ZEEKTAG)/zeek-$(ZEEKTAG).$(PLATFORM)-amd64.zip
```

```suggestion
		https://github.com/brimsec/zeek/releases/download/$(ZEEKTAG)/zeek-$(ZEEKTAG).$$(go env GOOS)-$$(go env GOARCH).zip
```

This will let you remove the PLATFORM stuff up above.

---

Beautiful